src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx

```diff
@@ -796,24 +798,61 @@ const MeasureGroups = (props: MeasureGroupProps) => {
                         type="B"
                         aria-label="Populations tab panel"
                         tabIndex={0}
-                        label={`Populations 
-                ${
-                  !!formik.errors.populations && activeTab !== "populations"
-                    ? "🚫"
-                    : ""
-                }`}
-                        data-testid="populations-tab"
-                        onClick={() => {
-                          setActiveTab("populations");
-                        }}
+                        label={
+                          <div>
+                            {Boolean(formik.errors.populations) ? (
+                              <ErrorIcon
+                                sx={{
+                                  color: "#AE1C1C",
+                                  marginRight: "10px",
+                                  height: "20px",
+                                  width: "20px",
+                                }}
+                              />
+                            ) : (
+                              <CheckCircleIcon
+                                sx={{
+                                  color: "#4D7E23",
+                                  marginRight: "10px",
+                                  height: "20px",
+                                  width: "20px",
+                                }}
+                              />
+                            )}
```

Comment:
as we are using these icons in multiple components, does it make sense to introduce separate components(like CompleteIndicator and IncompleteIndicator, name can be anything)?

Reply:
more usage- [src/components/editMeasure/details/EditMeasureDetailsSideNav.tsx](https://github.com/MeasureAuthoringTool/madie-measure/pull/737/files#diff-06f2fe6ff0e2aa831911a341f71005eb594366f9e54535d37088f8042d25d121)

Reply:
That's a good point. I moved that chunk into its own component